### PR TITLE
George - CHARMS PR

### PR DESCRIPTION
**Pull request by georgeb79**

### CODE REVIEW

**Review by alenaminsk — APPROVED** [review]

File: src/main/java/com/drivers/.DS_Store
Comment (iamdez99) [inline src/main/java/com/drivers/.DS_Store:1]: Please uncache this file

File: src/test/java/ServiceNow/CHARMS/Pages/MyRASHomePage.java
Comment (iamdez99) [inline src/test/java/ServiceNow/CHARMS/Pages/MyRASHomePage.java:1]: Remove space on line 6

File: src/test/java/ServiceNow/CHARMS/ScenariosData/IIQ_TestDataManager.java
Comment (iamdez99) [inline src/test/java/ServiceNow/CHARMS/ScenariosData/IIQ_TestDataManager.java:1]: Remove space line 6

File: src/test/java/ServiceNow/CHARMS/ScenariosData/RAS_Survey_TestDataManager.java
Comment (iamdez99) [inline src/test/java/ServiceNow/CHARMS/ScenariosData/RAS_Survey_TestDataManager.java:1]: Remove space line 6

File: src/test/java/ServiceNow/CHARMS/Steps/RAS_Screener_Submission_Steps.java
Comment (iamdez99) [inline src/test/java/ServiceNow/CHARMS/Steps/RAS_Screener_Submission_Steps.java:1]: There is a lot of hardcoding in this class that we should fix in this PR

**Review by iamdez99 — APPROVED** [review]
Reviewed and approved - juarezds

**Review by iamdez99 — APPROVED** [review]

**Review by matakevin — APPROVED** [review]
Reviewed comments and LGTM

**Pull request merged**